RecoCTPPS/TotemRPLocal/plugins/CTPPSLocalTrackLiteProducer.cc

```diff
@@ -101,4 +102,18 @@ CTPPSLocalTrackLiteProducer::produce( edm::Event& iEvent, const edm::EventSetup&
 
 //----------------------------------------------------------------------------------------------------
 
+void
+CTPPSLocalTrackLiteProducer::fillDescriptions( edm::ConfigurationDescriptions& descr )
+{
+  edm::ParameterSetDescription desc;
+
+  desc.add<edm::InputTag>( "tagSiStripTrack", edm::InputTag( "totemRPLocalTrackFitter" ) );
+  desc.add<edm::InputTag>( "tagDiamondTrack", edm::InputTag( "ctppsDiamondLocalTracks" ) );
+  desc.add<bool>( "doNothing", true ); // disable the module by default
+
+  descr.add( "ctppsLocalTrackLiteDefaultProducer", desc );
```

Comment:
This "Default" naming was introduced to generate this intermediate `cfi` as a "buffer" that is cloned (as `ctppsLocalTrackLiteProducer`), transformed, and inserted in the eras conditions [here](https://github.com/forthommel/cmssw/blob/45dc32c5ddfaa3a7a18b0192a5e79a9726ff6f06/RecoCTPPS/TotemRPLocal/python/ctppsLocalTrackLiteProducer_cfi.py).
Unless you have a better solution to have the eras feature add the `doNothing = cms.bool(False)` statement to his module when the `ctpps_2016` modifier is used?